users/validators.py

```
@@ -4,7 +4,12 @@
 from django.utils.deconstruct import deconstructible
 from django.utils.translation import gettext_lazy as _
 
-
+from django.core.exceptions import ValidationError
+from django.conf import settings
+from bs4 import BeautifulSoup
+from django.core.exceptions import ValidationError
```

Ruff (0.13.3)

Redefinition of unused `ValidationError` from line 7

Remove definition: `ValidationError`

(F811)